Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -598,8 +595,12 @@ public bool IsColliderValidGrabbable(Collider collider, bool ignoreCollidersNotI
 
                 if (!isValidGrabbable)
                 {
-                    // Remove it from the cache if the grabbable is no longer valid for the object
-                    componentCache.Remove(instanceId);
+                    if (currentGrabbable == null)
+                    {
+                        // Remove it from the cache if the grabbable is no longer valid for the object
+                        _ = componentCache.Remove(instanceId);
```

Comment:
why the `_  =` assignment here?

Reply:
Mostly just to acknowledge that this method returns _something_ but that we ignore it. It's mostly just syntax, but I finder it clearer for reading and knowing that this isn't just a `void` function.